what4/test/ExprBuilderSMTLib2.hs

```diff
@@ -1191,7 +1191,7 @@ testResolveSymBV searchStrat =
 main :: IO ()
 main = do
   testLevel <- TestLevel . fromMaybe "0" <$> lookupEnv "CI_TEST_LEVEL"
-  let solverNames = SolverName <$> [ "cvc4", "yices", "z3" ]
+  let solverNames = SolverName <$> [ "cvc4", "cvc5", "yices", "z3" ]
```

Comment:
While poking around at the test suites recently, I discovered that adding a new solver to the solver tests alone doesn't do much. In order to actually _use_ CVC5 in this test suite, you'd need to add some tests here:

https://github.com/GaloisInc/what4/blob/cff655cce036c70eb113a2f9b1d98dec8c88c2a4/what4/test/ExprBuilderSMTLib2.hs#L1311-L1313

Perhaps it would suffice to define `cvc5Tests = cvc4Tests` for now. If there are CVC5-specific test cases that we encounter later, we can always add on to `cvc5Tests` in the future.